style.css

```diff
@@ -531,3 +529,102 @@ footer {
   text-decoration: none;
   margin: 24px;
 }
+
+.poppins600 {
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+  color: #3c3a39;
+}
+
+.popup-container {
+  margin: 6% 7%;
+  padding: 24px;
+  border: 2px solid #ebecf0;
+  background-color: #fff;
+  position: fixed;
+  top: 100%;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
```

Comment:
Well done on making the popup interaction 💯 but make sure to also adjust the design, I noticed some differences with the Figma 👇🏽 

## Figma (Desktop)

![Screenshot 2023-04-13 at 13 55 58](https://user-images.githubusercontent.com/45739963/231844040-5c1a7554-3c8d-4c84-aab0-e7faf3d12bc3.png)

## Your design (Desktop)

![Screenshot 2023-04-13 at 13 57 32](https://user-images.githubusercontent.com/45739963/231844432-9f963ec6-c00e-4057-971a-ebf8482f80f4.png)

My suggested changes for above ☝🏽  :
 - The popup image should be taking `55%` of the width or something approximating.
 - The technologies font size needs to also be bigger.
 - Lastly, I'd suggest adding some more `Lorem Ipsum` to manage the space better, blank space makes the design look incomplete a bit. 🤔 

Some suggested changes for the mobile design as well 👇🏽 

## Figma (mobile)

![Screenshot 2023-04-13 at 14 04 01](https://user-images.githubusercontent.com/45739963/231845761-0f805fb7-ec7a-4a85-b7ae-aa43769aea11.png)

## Your design (mobile, before I scroll)

![Screenshot 2023-04-13 at 13 56 57](https://user-images.githubusercontent.com/45739963/231844313-f55215b6-1195-46ce-acb1-e7cf776a429f.png)

## Your design (mobile, after I scroll)

![Screenshot 2023-04-13 at 13 56 43](https://user-images.githubusercontent.com/45739963/231844236-863582c1-1dd8-40c1-b15a-82617103a17b.png)

The first thing is that it's impossible for me to see the action buttons because there's some issue with the scrolling, and also the text goes outside of the popup, make sure that the scrolling (if there needs to be one) happens inside of the popup and that I can see the whole design. Keep going with the design 🥂 you're almost there 🤓

Reply:
Ok thanks for your recommendations, It's done.